en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -94,184 +53,140 @@ Server).
+    cp $IS_HOME/repository/components/plugins/org.wso2.carbon.user.core_4.4.11.jar $WEB_APP_LIB
+    cp $IS_HOME/repository/components/plugins/org.wso2.carbon.user.api_4.4.11.jar $WEB_APP_LIB
+    cp $IS_HOME/repository/components/plugins/org.wso2.carbon.logging_4.4.11.jar $WEB_APP_LIB
+    cp $IS_HOME/repository/components/plugins/httpcore_4.3.3.wso2v1.jar $WEB_APP_LIB
+    cp $IS_HOME/repository/components/plugins/axis2_1.6.1.wso2v20.jar $WEB_APP_LIB
+    cp $IS_HOME/repository/components/plugins/opensaml_2.6.4.wso2v3.jar $WEB_APP_LIB
+    cp $IS_HOME/repository/components/plugins/jettison_1.3.4.wso2v1.jar $WEB_APP_LIB
+    cp $IS_HOME/lib/runtimes/cxf/javax.ws.rs-api-2.0-m10.jar $WEB_APP_LIB
+    cp $IS_HOME/lib/runtimes/cxf/cxf-bundle-2.7.16.wso2v1.jar $WEB_APP_LIB
+    cp $IS_HOME/lib/runtimes/cxf/neethi-3.0.3.jar $WEB_APP_LIB
+    cp $IS_HOME/lib/runtimes/cxf/wsdl4j-1.6.3.jar $WEB_APP_LIB
+    cp $IS_HOME/repository/components/plugins/commons-codec_1.4.0.wso2v1.jar
+    cp $IS_HOME/repository/components/plugins/commons-collections_3.2.2.wso2v1.jar
     ```
 
     !!! note
-        Make sure the WebApp container server (of endpoint apps) is
-        running with SSL enabled.
+        Make sure the WebApp container server (of endpoint apps) is running with SSL enabled.
     
-        e.g., if tomcat enabled the https connector, add the following to
-        `           catalina.sh          ` .
+        e.g., if tomcat enabled the https connector, add the following to `catalina.sh` .
     
-        ``` xml
-            JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.keyStore=$WEB_SERVER_KEYSTORE -Djavax.net.ssl.keyStorePassword=$password"
-            JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.trustStore=$WEBSERVER_TRUSTORE -Djavax.net.ssl.trustStorePassword=$password"
+        ``` java
+        JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.keyStore=$WEB_SERVER_KEYSTORE -Djavax.net.ssl.keyStorePassword=$password"
+        JAVA_OPTS="$JAVA_OPTS -Djavax.net.ssl.trustStore=$WEBSERVER_TRUSTORE -Djavax.net.ssl.trustStorePassword=$password"
         ```
     
 
 ## Running the sample
 
-1.  Download and install WSO2 IS and apache-tomcat into your local
-    machine. Let’s consider IS installation as
-    `           <IS_HOME>          ` and tomcat installation as
-    `           <TOMCAT_HOME>          `
+1.  Download and install WSO2 IS and apache-tomcat into your local machine. Let’s consider IS installation as `<IS_HOME>` and tomcat installation as
+    `<TOMCAT_HOME>`
 
 2.  Get the sample setup scripts from the following location:
-    `                     https://github.com/ayshsandu/samples/tree/master/is_samples/is_5.6.0/hosting-endpoints                   `
-    .
-3.  Open `           <TOMCAT_HOME>/conf/server.xml          ` file and
-    enable the https connector on 8443 port.
+    `https://github.com/ayshsandu/samples/tree/master/is_samples/is_5.6.0/hosting-endpoints`.
```

Comment:
Change this personal repo reference to WSO2 reference `https://github.com/wso2/samples-is/blob/master/host-endpoints-externally`